tests/vmidefaults_test.go

```diff
@@ -220,4 +221,53 @@ var _ = Describe("[Serial][sig-compute]VMIDefaults", func() {
 
 	})
 
+	Context("Input defaults", func() {
+
+		It("[test_id:TODO]Should be set in VirtualMachineInstance", func() {
+
+			By("Creating a VirtualMachineInstance with an input device without a bus or type set")
+			vmi = tests.NewRandomVMI()
+			vmi.Spec.Domain.Devices.Inputs = append(vmi.Spec.Domain.Devices.Inputs, v1.Input{
+				Name: "foo-1",
+			})
+
+			vmi, err = virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Create(vmi)
+			Expect(err).ToNot(HaveOccurred())
+
+			By("Waiting for successful start")
+			tests.WaitForSuccessfulVMIStart(vmi)
+
```

Comment:
Is it required to start the VM, or might there be a more lightweight way to tests the autoattach?

Reply:
I guess for this specific test it isn't required as this is just asserting the behaviour of the VirtualMachineInstance mutation webhook when applying defaults. I'll drop the start.